pkg/cmd/generate.go

```diff
@@ -58,6 +59,8 @@ func Generate(paths []string) error {
 				return errors.Wrap(err, "failed to marshal object")
 			}
 
+			validateOutput(data)
```

Comment:
validation is done only for `generate`?
Any reason why not doing it also for `apply` and `create`?